core/templates/pages/admin-page/roles-tab/role-graph.directive.html

```diff
@@ -234,15 +234,18 @@
           <legend class="text-center">Remove Reviewer From Community</legend>
           <div class="d-xl-inline-flex form-group">
             <label for="label-target-form-reviewer-username" class="col-md-4 col-lg-4 col-sm-4 col-form-label">Enter Username</label>
-            <div class="col-md-4 col-lg-4 col-sm-4 ml-auto">
+            <div class="col-md-8 col-lg-8 col-sm-8 ml-auto">
               <input type="text"
                      id="label-target-form-reviewer-username"
                      class="form-control protractor-test-form-reviewer-username"
                      placeholder="Enter username"
                      ng-model="$ctrl.formData.removeContributionReviewer.username">
             </div>
+          </div>
+
+          <div class="d-xl-inline-flex form-group">
             <label for="label-target-form-remove-reviewer" class="col-md-4 col-lg-4 col-sm-4 col-form-label">Select Rights</label>
-            <div class="col-md-4 ml-auto">
+            <div class="col-md-8 col-lg-8 col-sm-8 ml-auto">
```

Comment:
Hm, should it be `col-*-3` here and ditto for the other sibling elements above?

Reply:
Hey @kevintab95 With the current changes, the UI looks like 
![Screenshot from 2020-10-11 10-51-07](https://user-images.githubusercontent.com/44888949/95707406-15201e00-0c77-11eb-9324-3d471fc83bcc.png)

which seems to be consistent but If I make the changes suggested by you, I am getting something like 
![Screenshot from 2020-10-12 10-32-55](https://user-images.githubusercontent.com/44888949/95707381-020d4e00-0c77-11eb-9f15-8d526f71b9a6.png)

Do we want both the options to appear in a single row? Please let me know.

Reply:
Hi @Kajol-Kumari, hm, that's weird, I had expected my suggested changes to look like this (tried this locally):
![image](https://user-images.githubusercontent.com/11008603/95709207-cf198900-0c7b-11eb-9714-4e867ee310ff.png)

![image](https://user-images.githubusercontent.com/11008603/95709388-2f102f80-0c7c-11eb-8468-9b6444d88876.png)


I had assumed this was the intended design because we had used an "inline" display class in the parent div.

But looking at the screenshots, I lean towards having the inputs be one below the other (like in your first screenshot). Thanks!

Reply:
Thanks for clarifying!